index.html

```diff
@@ -0,0 +1,416 @@
+                Classical physics: Newtonian mechanics 
+            </p>
+        </div>
+        <div class="deck deck--desktop-screen">
+            <div class="card card--no-padding">
+                <img src="./assets/teammate-1.png" alt="" class="card__image">
+                <div class="card__bio">
+                    <p class="card__name card__name--black card__name--strengthened">Ben Johnson</p>
+                    <p class="card__role">Consultant</p>
+                    <div class="socials">
+                        <img src="./assets/facebook-icon.png" alt="" class="social">
+                        <img src="./assets/instagram-icon.png" alt="" class="social">
+                        <img src="./assets/twitter-icon.png" alt="" class="social">
+                    </div>
+                </div>
+            </div>
+            <div class="card card--no-padding">
+                <img src="./assets/teammate-2.png" alt="" class="card__image">
+                <div class="card__bio">
+                    <p class="card__name card__name--black card__name--strengthened">Ben Johnson</p>
+                    <p class="card__role">Consultant</p>
+                    <div class="socials">
+                        <img src="./assets/facebook-icon.png" alt="" class="social">
+                        <img src="./assets/instagram-icon.png" alt="" class="social">
+                        <img src="./assets/twitter-icon.png" alt="" class="social">
+                    </div>
+                </div>
+            </div>
+            <div class="card card--no-padding">
+                <img src="./assets/teammate-1.png" alt="" class="card__image">
+                <div class="card__bio">
+                    <p class="card__name card__name--black card__name--strengthened">Ben Johnson</p>
+                    <p class="card__role">Consultant</p>
+                    <div class="socials">
+                        <img src="./assets/facebook-icon.png" alt="" class="social">
+                        <img src="./assets/instagram-icon.png" alt="" class="social">
+                        <img src="./assets/twitter-icon.png" alt="" class="social">
+                    </div>
+                </div>
+            </div>
+        </div>                  
+    </section>
+    <section class="form-cta">
+        <form class="form form--desktop-screen">
+            <div class="note">
+                <h2 class="note__subtitle">Contact Us</h2>
+                <h1 class="note__title note__title--left note__title--text-big">
+                    Make an <br>
+                    Appointment
+                </h1>
+            </div>
+            <div class="form__inputs form__inputs__desktop-screen">
+                <input class="form__field" type="text" placeholder="Full Name">
+                <input class="form__field" type="text" placeholder="example@gmail.com">
+            </div>
+            <div class="form__selects form__selects__desktop-screen">
```

Comment:
`form__selects__desktop-screen` is not a valid BEM class. Two Elements on the same class is not allowed.